plugins/FakeCardReader.hpp

```diff
@@ -65,6 +65,11 @@ class FakeCardReader : public dunedaq::appfwk::DAQModule
 
   void generate_data(sink_t* queue, int link_id);
   void generate_tp_data(tp_sink_t* queue, int link_id);
+  void generate_tp_data_v0(tp_sink_t* queue, int link_id);
+
+  void* m_data_{nullptr};
```

Comment:
m_data as a member field for this Module is as vague as it can get..
Please use m_tp_data_buffer or similar.